### PR TITLE
Update Mock Response doc

### PR DESCRIPTION
**Pull request by Yubee116**
### Preview Link
https://deploy-preview-3347--tyk-docs.netlify.app/docs/nightly/advanced-configuration/transform-traffic/endpoint-designer/#mock-response
<br>

### Description
Added more info to existing text
Updated existing text grammar
Modified existing text to list

Relates to TT-9720

## Checklist
- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md [hunk 1]
```diff
@@ -157,6 +157,8 @@ This plugin allows you to change the method of a request. See [Method Transforms
 
 This plugin allows you to mock responses for an API endpoint. This can be useful when creating a new API, or when making a development API available to an external team.
 
+Mocked endpoints will not be authenticated, will not process other middleware configured in the API, and will have no analytics.
```
Comment (dcs3spp) [inline tyk-docs/content/advanced-configuration/transform-traffic/endpoint-designer.md]:
```suggestion
Mocked endpoints will not be authenticated, will not process other middleware configured in the API and will have no analytics.
```

**Review by dcs3spp — APPROVED** [review]
lgtm @Yubee116 I have approved changes. Once the PR has been peer reviewed we will merge and release.

**Review by OldStubbsy — APPROVED** [review]
looks good!

**Pull request merged**